pkg/certificate/certificate.go

```diff
@@ -46,11 +46,12 @@ func NewKeyPair(tlsGenerate bool, keyFile, certFile string) *Keypair {
 	}
 }
 
-func (k *Keypair) generate() error {
-	org := ""
+func CreateRawKeyPair() (bytes.Buffer, bytes.Buffer, error) {
+	var cert, key bytes.Buffer
+	org := "VMware"
 	priv, err := rsa.GenerateKey(rand.Reader, 2048)
 	if err != nil {
-		return err
+		return cert, key, err
```

Comment:
Go actually doesn't allow this. I had it initially and it gave me a type error. Am I missing something @caglar10ur ?